etc/wpa_supplicant/wpa_supplicant.conf

```diff
@@ -1,10 +1,5 @@
-# Please change to match your country code
-country=US
-update_config=1
-ctrl_interface=/var/run/wpa_supplicant
```

Comment:
This is necessary on Raspbian, are you sure it's not on Alpine?

Reply:
Its not necessary on alpine